docs/egress.md

```diff
@@ -65,8 +65,8 @@ data:
 A typical Egress resource example:
 
 ```yaml
-apiVersion: crd.antrea.io/v1alpha2
 kind: Egress
+apiVersion: crd.antrea.io/v1alpha2
```

Comment:
nit: why revert `kind` and `apiVersion`'s order? all K8s examples put `apiVersion` in the first line, and it's the default order when getting one resource's yaml via kubectl. 
https://kubernetes.io/docs/concepts/services-networking/service/#defining-a-service
https://kubernetes.io/docs/concepts/services-networking/network-policies/#networkpolicy-resource